### PR TITLE
Added functionality for manually adding hired participants

### PR DESCRIPTION
**Pull request by Zenen-freshworks**
Also added associated cypress and npm tests

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -137,6 +139,45 @@ app.post(`${apiBaseUrl}/employer-actions`,
...
+      const user = req.hcapUserInfo;
+      const { participantInfo } = req.body;
+      participantInfo.preferredLocation = user.regions[0];
+      participantInfo.crcClear = 'yes';
+      participantInfo.interested = 'yes';
+
+      const response = await makeParticipant(participantInfo);
+      await setParticipantStatus(user.id, response.id, 'prospecting');
+      await setParticipantStatus(user.id, response.id, 'interviewing', { contacted_at: participantInfo.contactedDate });
+      console.log(participantInfo.contactedDate);
```
Comment (mauriciofreshw) [inline server/server.js:157]: committed console.log

Comment (Zenen-freshworks) [reply server/server.js]: ! fixed

**Review by mauriciofreshw — APPROVED** [review]

**Review by mauriciofreshw — DISMISSED** [review]

**Review by ashtonmeuser — DISMISSED** [review]

**Comment by ashtonmeuser** [comment]
Tested locally. Noticed a few things:
- Not all error messages appear under required fields when hitting submit on blank/partially-filled modal form
- Future dates are accepted for date first contacted and date offer accepted

**Review by mauriciofreshw — APPROVED** [review]

File: server/tests/participant.form.test.js [hunk 2]
```diff
@@ -478,4 +478,36 @@ describe('Participants Service', () => {
     participants = await getParticipants({ isMoH: true });
     expect(participants.data[0].statusInfo).toEqual('Hired');
   });
+
+  it('checks the status of an entry made through the new-hired-participant endpoint', async () => {
```
Comment (ashtonmeuser) [inline server/tests/participant.form.test.js:482]: This test should have caught HCAP-361.

File: server/server.js [hunk 3]
```diff
@@ -137,6 +139,44 @@ app.post(`${apiBaseUrl}/employer-actions`,
...
+      participantInfo.preferredLocation = user.regions[0];
+      participantInfo.crcClear = 'yes';
+      participantInfo.interested = 'yes';
+
+      const response = await makeParticipant(participantInfo);
+      await setParticipantStatus(user.id, response.id, 'prospecting');
+      await setParticipantStatus(user.id, response.id, 'interviewing', { contacted_at: participantInfo.contactedDate });
+      await setParticipantStatus(user.id, response.id, 'offer_made');
+      await setParticipantStatus(user.id, response.id, 'hired', {
+        nonHcapOpportunity: participantInfo.nonHcapOpportunity,
```
Comment (ashtonmeuser) [inline server/server.js:159]: Site should be recorded here.

**Pull request merged**